mne_bids/write.py

```diff
@@ -331,10 +331,21 @@ def _scans_tsv(raw, raw_fname, fname, overwrite=False, verbose=True):
     """
     # get measurement date from the data info
     meas_date = raw.info['meas_date']
-    if isinstance(meas_date, (tuple, list, np.ndarray)):
+    if meas_date is None:
+        acq_time = 'n/a'
+    elif isinstance(meas_date, (tuple, list, np.ndarray)):  # noqa: E501
+        # for MNE < v0.20
         acq_time = _stamp_to_dt(meas_date).strftime('%Y-%m-%dT%H:%M:%S')
+    elif isinstance(meas_date, datetime):
+        # for MNE >= v0.20
+        acq_time = meas_date.strftime('%Y-%m-%dT%H:%M:%S')
     else:
-        acq_time = 'n/a'
+        # this should not be reached since MNE makes checks
+        acq_time = 'n/a'  # noqa: E501 F821
+        warn("We have checked everything "   # noqa: E501 F821
+             "that the acquisition time could be ... "
+             "we have to set it to 'n/a', because "
+             "there is no other sensible thing to do... ")
```

Comment:
I'd in fact suggest removing this else part if it's never reached because it reduces the test coverage.